src/hyperloglog.c

```diff
@@ -394,6 +394,25 @@ static char *invalid_hll_err = "-INVALIDOBJ Corrupted HLL object detected";
     } while (0)
 #define HLL_ALPHA_INF 0.721347520444481703680 /* constant for 0.5/ln(2) */
 
+static inline int32_t count_trailing_zeros_64(uint64_t value) {
+#if defined(__clang__) || defined(__GNUC__)
+    return __builtin_ctzll(value);
+#elif defined(_MSC_VER) && defined(_WIN64)
+    uint32_t trailing_zero = 0;
+    _BitScanForward64(&trailing_zero, value);
+    return trailing_zero;
+#else
```

Comment:
```suggestion
#else
```
We don't need to support windows. I saw the other point about how we use `__builtin_clzl` which is supported in basically all the same places as __builtin_clzll, so it should be safe to just use __builtin_clzll until someone complains. I'm okay either way.

Reply:
Removed now